packages/database/src/modelClasses/Entity.js

```diff
@@ -24,6 +24,8 @@ const WORLD = 'world';
 const PROJECT = 'project';
 const CITY = 'city';
 
+// Note: if a new type is not included in `ORG_UNIT_ENTITY_TYPES`,
```

Comment:
Hi sorry what does this mean?

Reply:
I hoped that the comment would be self-explanatory... If it isn't, perhaps I need to change it 🙂

Reply:
It's related to

https://github.com/beyondessential/tupaia-backlog/issues/2115#issuecomment-780237062
https://github.com/beyondessential/tupaia-backlog/issues/2122#issuecomment-780191494

which potentially caused this issue yesterday: https://beyondessential.slack.com/archives/G01CW3WKFPS/p1613515567002300 (although it's hard to confirm)

Reply:
Ah very useful info, thanks! As mentioned in chat maybe good to put a link in here to some docs about it